cmd/system-probe/check_cmd.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 var (
+	//nolint:unused
```

Comment:
For understanding: Any reason we don't just delete unused code? Is there like a procedure for this where we delete it after some time?

Reply:
I'm not removing the unused code yet because I need to understand why this hadn't been removed before cleaning up. In general we _should_ remove unused code whenever possible.